packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -688,9 +684,9 @@ export class ColorPicker
 
   async componentWillLoad(): Promise<void> {
     setUpLoadableComponent(this);
-
-    const { allowNullValues, color, format, value } = this;
-    const willSetNoColor = allowNullValues && !value;
+    this.handleEmptyValues();
+    const { isClearable, color, format, value } = this;
```

Comment:
Can you add a new line here to visually separate method calls from var assignment?